mailprocessing/processor/imap.py

```diff
@@ -344,6 +344,8 @@ def _load_cache(self):
         """
         This method loads a previously stored header cache from the cache file.
         """
+        if self.cache_file is None:
+            return
```

Comment:
`self.cache_file` is `None` if we don't have a cache file specified and we're using in-memory cache only.